components/user_profile/index.ts

```diff
@@ -13,14 +13,11 @@ type OwnProps = {
     userId: string;
 }
 
-function makeMapStateToProps() {
+function makeMapStateToProps(state: GlobalState, ownProps: OwnProps) {
     const getDisplayName = makeGetDisplayName();
-
-    return (state: GlobalState, ownProps: OwnProps) => {
-        return {
-            displayName: getDisplayName(state, ownProps.userId, true),
-            user: getUser(state, ownProps.userId),
-        };
+    return {
```

Comment:
Removing the callback return means `makeGetDisplayName` is created from scratch every time, breaking the intended memoization here, and we should just use the underlying, unmemoized selector instead. Or was this change intentional?

Reply:
Unintentional. I think I forgot to undo some experimentation I was doing with `useSelector`+`createSelector` and the make-selector pattern re: `makeGetDisplayName`. I'll revert